include/atama/head/head.hpp

```diff
@@ -18,54 +18,59 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-#ifndef ATAMA__HEAD_HPP_
-#define ATAMA__HEAD_HPP_
+#ifndef ATAMA__HEAD__HEAD_HPP_
+#define ATAMA__HEAD__HEAD_HPP_
 
-#include <aruku/walking.hpp>
-#include <kansei/imu.hpp>
-#include <keisan/geometry/point_2.hpp>
-#include <robocup_client/robocup_client.hpp>
+#include <math.h>
+#include <time.h>
 
 #include <memory>
 #include <string>
 #include <vector>
 
-namespace atama
+#include "keisan/geometry/point_2.hpp"
+#include "ninshiki_interfaces/msg/detected_object.hpp"
+#include "ninshiki_interfaces/msg/detected_objects.hpp"
+#include "tachimawari/joint/model/joint.hpp"
+
+namespace atama::head
 {
 
 class Head
 {
 public:
   enum
   {
-    SCAN_BALL_UP                = 0,
-    SCAN_BALL_DOWN              = 1,
-    SCAN_VERTICAL               = 2,
-    SCAN_HORIZONTAL             = 3,
-    SCAN_MARATHON               = 4
+    SCAN_UP                = 0,
+    SCAN_DOWN              = 1,
+    SCAN_VERTICAL          = 2,
+    SCAN_HORIZONTAL        = 3,
+    SCAN_MARATHON          = 4,
+    SCAN_CUSTOM            = 5,
+    TRACK_OBJECT           = 6,
+    MOVE_BY_ANGLE          = 7,
+    LOOK_TO_POSITION       = 8,
   };
+  std::vector<ninshiki_interfaces::msg::DetectedObject> detection_result;
 
   int marathon_index;
 
-  Head(std::shared_ptr<aruku::Walking> walking, std::shared_ptr<kansei::Imu> imu);
+  Head(int camera_width, int camera_height);
```

Comment:
I think the camera size should be handled by ninshiki

Reply:
done